fuse_optimizers/src/batch_optimizer.cpp

```diff
@@ -202,6 +202,11 @@ void BatchOptimizer::optimizationLoop()
 
 void BatchOptimizer::optimizerTimerCallback(const ros::TimerEvent& event)
 {
+  // If an "ignition" transaction hasn't been received, then we can't do anything yet.
+  if (!started_)
```

Comment:
It was guarded in one place, but not both...
https://github.com/locusrobotics/fuse/blob/2cdbbfba815e585104ba4f3457462853564fba9b/fuse_optimizers/src/batch_optimizer.cpp#L268